integration-tests/lra/pom.xml

```diff
@@ -47,6 +47,10 @@
             <groupId>org.apache.camel.quarkus</groupId>
             <artifactId>camel-quarkus-lra</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.camel.quarkus</groupId>
+            <artifactId>camel-quarkus-mock</artifactId>
```

Comment:
Please also add to the `virtualDependencies` profile.

Reply:
Nice catch!

Reply:
When I run `mvn process-resources -Pformat -N` to regenerate.
```
[ERROR] Failed to execute goal org.l2x6.cq:cq-maven-plugin:4.4.7:format (sort-poms) on project camel-quarkus: Unable to parse configuration of mojo org.l2x6.cq:cq-maven-plugin:4.4.7:format for parameter updateVirtualDependencies: Cannot set 'updateVirtualDependencies' in class org.l2x6.cq.maven.FormatPomsMojo: Type org.apache.maven.shared.utils.io.DirectoryScanner not present -> [Help 1]

```

@ppalaga is it an issue with cq-maven-plugin?